Example/ContactsListSettingsTableViewController.swift

```diff
@@ -0,0 +1,24 @@
+import UIKit
+import MiniApp
+
+class ContactsListSettingsTableViewController: UITableViewController {
```

Comment:
I think the contacts should be editable too, right? We can automatically populate the list with random UUIDs and then allow them to add/delete items.

Reply:
Right now there are like 10 dummy contacts added to the list (associated with random UUID's). Right now we don't allow the user to edit/add the contacts. Do you want to allow the user to add/delete the items from the list ?? Deleting the item from the list wouldn't be a big functionality though, but adding a new item to the list needs some little effort

Reply:
Done. Right now you can swipe to delete the contact from the list and added a + button in the right top corner. When + button is tapped, it will automatically add a new UUID list in the list.

Reply:
<img src=https://user-images.githubusercontent.com/58416454/90661375-168a2700-e282-11ea-859e-2e5fa1b56e73.png width="240" height="500">

Reply:
Sorry, I wasn't very clear on this. The IDs that are passed by the Host App won't be UUIDs, they will be some sort of custom ID scheme. QA needs to be able to test a variety of different inputs in the contact list.

So on this screen, when the user taps the "+" button, it should allow them to input the ID they want to add.

Reply:
Understood 👍 , I will change that too. But the list which displays this UUID is fine right? Also would like to know if these items in the list (random UUIDs and the value added by the QA) should persist?